vocabularies/Analytics.json

```diff
@@ -161,6 +161,102 @@
+            "@Core.Description": "Collection of aggregation levels forming a leveled hierarchy",
+            "@Core.LongDescription": "Each element in the collection defines the property names for one level.\n            A property must not referenced by more than one level.\n            The first element in the collection defines the property names of the coarsest level,\n            the following elements define the property names of consecutively finer-grained aggregation levels.\n            All referenced properties must be groupable."
+          },
+          {
+            "$Name": "ExpandLevel",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number of levels to be expanded, counting from left",
+            "@Core.LongDescription": "The function result comprises the leveled hierarchy with the expanded levels and the following level as leaves in preorder."
+          },
+          {
+            "$Name": "Aggregation",
+            "$Collection": true,
+            "@Core.Description": "Properties to aggregate for all result entries on all levels",
+            "@Core.LongDescription": "All properties in this collection must be custom aggregates."
+          },
+          {
+            "$Name": "SiblingOrder",
+            "$Collection": true,
+            "$Type": "Analytics.AutoExpandSiblingOrder",
+            "@Core.Description": "Sort specification to apply to all direct descendants of a given entry in the resulting leveled hierarchy"
+          },
+          {
+            "$Name": "BeforeAggregationFilter",
+            "@Core.Description": "Expression valid for `filter` transformation to restrict the input set before any further procressing",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "AggregatedValuesLeafFilter",
+            "@Core.Description": "Expression valid for `filter` transformation to restrict the input set at the most detailed grouping level with conditions on aggregated values",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "Skip",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number of entries to skip from the top of the fully ordered result",
+            "@Core.OptionalParameter": { "DefaultValue": "0" }
+          },
+          {
+            "$Name": "Top",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number of entries to return from the result set after any skipping (absent means all)",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "ResultEntriesCount",
+            "$Type": "Edm.Bool",
+            "@Core.Description": "Whether to return the total number of entries in the result independent of Skip/Top",
+            "@Core.OptionalParameter": { "DefaultValue": "false" }
+          }
+        ],
+        "$ReturnType": { "$Collection": true, "$Type": "Edm.EntityType" }
```

Comment:
I guess the EntityType is not enough. We will need some of the information that we also use in a hierarchical query. Most importantly, the [LimitedDescendantCount](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L109:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22) but also the [DrillState](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L112:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22) and the [DistanceFromRoot](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L121:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22).

Reply:
So we need an instance annotation whose type contains a subset of the properties of `Hierarchy.RecursiveHierarchyType`. Which subset exactly?

Reply:
@ThomasChadzelek I had found the [LimitedDescendantCount](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L109:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22), the [DrillState](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L112:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22) and the [DistanceFromRoot](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L121:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22). Is that it? Or already to much?

Reply:
Is it sufficient if these instance annotations are returned by the auto-expand function? If one node is expanded, they must be computed by the OData model itself.

Reply:
I think yes. In the other cases, the model is creating requests to manually expand the visual groups. The backend would not even be aware that we are creating a visual grouping on the screen.

Reply:
> @ThomasChadzelek I had found the [LimitedDescendantCount](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L109:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22), the [DrillState](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L112:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22) and the [DistanceFromRoot](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Hierarchy.xml#L121:~:text=%3CComplexType%20Name=%22-,RecursiveHierarchyType,-%22). Is that it? Or already too much?

What about LimitedRank? Our new favorite ;-)

DistanceFromRoot might be redundant because the hierarchy is leveled, not recursive?